crates/sui-core/src/authority_server.rs

```diff
@@ -194,46 +193,62 @@ impl ValidatorServiceMetrics {
                 registry,
             )
             .unwrap(),
-            tx_verification_latency: MystenHistogram::new_in_registry(
+            tx_verification_latency: register_histogram_with_registry!(
                 "validator_service_tx_verification_latency",
                 "Latency of verifying a transaction",
+                mysten_metrics::SUBSECOND_LATENCY_SEC_BUCKETS.to_vec(),
                 registry,
-            ),
-            cert_verification_latency: MystenHistogram::new_in_registry(
+            )
+            .unwrap(),
+            cert_verification_latency: register_histogram_with_registry!(
                 "validator_service_cert_verification_latency",
                 "Latency of verifying a certificate",
+                mysten_metrics::SUBSECOND_LATENCY_SEC_BUCKETS.to_vec(),
                 registry,
-            ),
-            consensus_latency: MystenHistogram::new_in_registry(
+            )
+            .unwrap(),
+            consensus_latency: register_histogram_with_registry!(
                 "validator_service_consensus_latency",
                 "Time spent between submitting a shared obj txn to consensus and getting result",
```

Comment:
Since you are here, maybe update the comment to "... and getting acknowledgement".